testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/managedbean/annotation/AnnotatedManagedBeanTestCase.java

```diff
@@ -37,6 +40,11 @@
 @RunWith(Arquillian.class)
 public class AnnotatedManagedBeanTestCase {
 
+    @BeforeClass
+    public static void beforeClass() {
+        assumeTrue(System.getProperty("ts.ee9") == null);
```

Comment:
Also ts.bootable.ee9

Reply:
Noted. Thanks. FWIW, I stole that method from existing classes. This kind of thing belongs in a test utility somewhere, but my quick scan didn't find one.

Reply:
Found in testsuite/shared. Extracted updated. Changes inbound.